ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/server/api/validation/DeduplicationPeriodValidator.scala

```diff
@@ -48,7 +48,11 @@ class DeduplicationPeriodValidator(
   ): Either[StatusRuntimeException, Duration] = if (duration.isNegative)
     Left(
       errorFactories
-        .invalidField(fieldName, "Duration must be positive", definiteAnswer = Some(false))
+        .invalidField(
```

Comment:
Nit: is it the auto-format?

Reply:
Seems to be so, reformatting doesn't change anything.

Reply:
Yes but subsequent reformats won't revert the change.